packages/nextlove/src/with-route-spec/middlewares/with-validation.ts

```diff
@@ -292,12 +292,12 @@ export const withValidation =
 
       const isFormData = Boolean(input.formData)
 
-      if (isFormData && willValidateRequestBody) {
-        req.body = input.formData?.parse(req.body)
+      if (isFormData && willValidateRequestBody && input.formData) {
```

Comment:
Can `isFormData` contain the check for non-empty form data?

Reply:
Yes, that would still work, the issue is that because of the `?` conditional thing, when formData is undefined, we overwrite the req.body to undefined too.

Reply:
I think if you do ` const isFormData = Boolean(input.formData) && input.formData` then you can still remove the `.?`